Include/pymath.h

```diff
@@ -215,4 +215,16 @@ PyAPI_FUNC(void) _Py_set_387controlword(unsigned short);
 					 (X) == -Py_HUGE_VAL))
 #endif
 
+/* Return whether integral type *type* is signed or not. */
+#define _Py_IntegralSigned(type) ((type)(-1) < 0)
```

Comment:
I suggest to put "Type" in the macro names. Maybe _Py_IsTypeSigned(type), _Py_TypeMax(), etc.?

Reply:
sg